Sources/CreateProject/DataFactory.swift

```diff
@@ -166,25 +166,17 @@ enum DataFactory {
 
         [libraries]
         # web is not actually functional but required to use a Web export template when creating the .pck file via `make pack`
-        web.debug = "res://bin/lib\(projectName).\(dynamicExtension)"
-        web.release = "res://bin/lib\(projectName).\(dynamicExtension)"
-
-        macos.debug = "res://bin/lib\(projectName).\(dynamicExtension)"
-        macos.release = "res://bin/lib\(projectName).\(dynamicExtension)"
-        windows.debug.x86_32 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        windows.release.x86_32 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        windows.debug.x86_64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        windows.release.x86_64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        linux.debug.x86_64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        linux.release.x86_64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        linux.debug.arm64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        linux.release.arm64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        linux.debug.rv64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        linux.release.rv64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        android.debug.x86_64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        android.release.x86_64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        android.debug.arm64 = "res://bin/lib\(projectName).\(dynamicExtension)"
-        android.release.arm64 = "res://bin/lib\(projectName).\(dynamicExtension)"
+        web.debug = "res://bin/lib\(projectName).so"
+        web.release = "res://bin/lib\(projectName).so"
+
+        macos.debug = "res://bin/lib\(projectName).dylib"
+        macos.release = "res://bin/lib\(projectName).dylib"
+        windows.debug = "res://bin/lib\(projectName).dll"
+        windows.release = "res://bin/lib\(projectName).dll"
+        linux.debug = "res://bin/lib\(projectName).so"
+        linux.release = "res://bin/lib\(projectName).so"
+        android.debug = "res://bin/lib\(projectName).so"
+        android.release = "res://bin/lib\(projectName).so"
```

Comment:
assuming a dynamic library compilation, the file types for the various platforms are known.
- windows: .dll
- linux: .so
- android: .so
- mac: .dylib
- ios: .dylib
```suggestion
        web.debug = "res://bin/lib\(projectName).so"
        web.release = "res://bin/lib\(projectName).so"

        macos.debug = "res://bin/lib\(projectName).dylib"
        macos.release = "res://bin/lib\(projectName).dylib"
        ios.debug = "res://bin/lib\(projectName).dylib"
        ios.release = "res://bin/lib\(projectName).dylib"
        windows.debug = "res://bin/lib\(projectName).dll"
        windows.release = "res://bin/lib\(projectName).dll"
        linux.debug = "res://bin/lib\(projectName).so"
        linux.release = "res://bin/lib\(projectName).so"
        android.debug = "res://bin/lib\(projectName).so"
        android.release = "res://bin/lib\(projectName).so"
```

Reply:
this doesn't account for different CPUs though 🤔

Reply:
it seems like a complex Makefile is needed to truly support multiple platforms, for now it's an exercise for the developer...

Reply:
This noes not take debug/release into account. Maybe add a debug/release subfolder? My dll paths on Windows look like this:

`C:\Code\Swift\GoSwiftDot2\.build\x86_64-unknown-windows-msvc\debug\GoSwiftDot2.dll`.
`C:\Code\Swift\GoSwiftDot2\.build\x86_64-unknown-windows-msvc\release\GoSwiftDot2.dll`

Reply:
Great point.